app/helpers/layout_helper.rb

```diff
@@ -158,6 +158,10 @@ def per_page(collection)
     [Setting[:entries_per_page], collection.total_entries].min
   end
 
+  def results_info(options = {})
```

Comment:
Maybe put this in the partial instead of a helper? seems it would be simpler in HTML.